packages/orbit-components/src/Textarea/README.md

```diff
@@ -16,32 +16,34 @@ After adding import into your project you can use it simply like:
-| onFocus      | `event => void \| Promise` |              | Function for handling onFocus event.                                                                           |
-| onBlur       | `event => void \| Promise` |              | Function for handling onBlur event.                                                                            |
-| placeholder  | `TranslationString`        |              | The placeholder of the Textarea.                                                                               |
-| ref          | `func`                     |              | Prop for forwarded ref of the Textarea. [See Functional specs](#functional-specs)                              |
-| resize       | [`enum`](#enum)            | `"vertical"` | The resize option for Textarea.                                                                                |
-| rows         | `number`                   |              | Specifies the height of the text area (in lines).                                                              |
-| readOnly     | `boolean`                  |              | Adds readOnly to html textarea element.                                                                        |
-| required     | `boolean`                  |              | If true, the label is displayed as required.                                                                   |
-| spaceAfter   | `enum`                     |              | Additional `margin-bottom` after component.                                                                    |
-| tabIndex     | `string \| number`         |              | Specifies the tab order of an element                                                                          |
-| value        | `string`                   |              | Specifies the value of the Textarea. To be used with controlled usage.                                         |
-| helpClosable | `boolean`                  | `true`       | Whether to display help as a closable tooltip, or have it open only while the field is focused, same as error. |
+| Name           | Type                       | Default      | Description                                                                                                    |
+| :------------- | :------------------------- | :----------- | :------------------------------------------------------------------------------------------------------------- |
+| dataAttrs      | `Object`                   |              | Optional prop for passing `data-*` attributes to the `textarea` DOM element.                                   |
+| dataTest       | `string`                   |              | Optional prop for testing purposes.                                                                            |
+| id             | `string`                   |              | Set `id` for `Textarea`                                                                                        |
```

Comment:
`.` is missing

Reply:
I wasn't paying much attention to the README file as there is separate task to add documentation plus "Check if remaining docs (README file) includes all important information.".

But sure, I'll fix these things you pointed out.